functions/__fzf_search_history.fish

```diff
@@ -4,7 +4,9 @@ function __fzf_search_history --description "Search command history. Replace the
     set command_with_ts (
         # Reference https://devhints.io/strftime to understand strftime format symbols
         builtin history --null --show-time="%m-%d %H:%M:%S | " |
-        fzf --read0 --tiebreak=index --query=(commandline) |
+        fzf --read0 --tiebreak=index --query=(commandline) \
+            --preview="echo {4..}" \
+            --preview-window="bottom:10%" |
```

Comment:
I think you can increase this to 20%.
I'm trying to decide if locking it in at the bottom is a problem for people who customize FZF default opts to not use --reverse.

Reply:
I don't see why it would be a problem for people not using `--reverse`. The preview isn't linked to the way the results are displayed.